react-debugging.html

```diff
@@ -62,15 +62,17 @@
           }
           // No need to change anything ABOVE this line ^
 
+          //Solutions: 
+          // 1. Moved <div style={handStyles}> above header 
```

Comment:
While this will work, more traditionally when looking to return more than one line in React, you need React fragments although a div will also do the trick as well.